README.md

```diff
@@ -1,4 +1,355 @@
+
+
+### Displaying metrics
+
+1. We navigate into `microservice-one/src/main/java/application/rest` and create a new file `JaxrsManager.java`.
+2. Add the required imports
+```
+package application.rest;
+
+import org.eclipse.microprofile.metrics.Metadata;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.MetricType;
+import org.eclipse.microprofile.metrics.MetricUnits;
+import org.eclipse.microprofile.metrics.Timer;
+import javax.json.JsonObject;
+import javax.json.Json;
+```
+
+3. Now we need to paste the code below into the file in order to initialize the registry and the metadata that we are using to record the metrics for the microservice in the next steps.
+```
+public class JaxrsManager {
+
+  private MetricRegistry registry;
+  private Timer testDataCalcTimer;
+  
+  public JaxrsManager() {
+    //this.jaxrsManager = jaxrsManager;
+    registry = JaxrsApplication.registry;
+    setupMetrics();       
+  }
+    
+  private void setupMetrics() {
+    // Timer
+    Metadata testDataCalcTimerMetadata = new Metadata(
+          "testDataCalcTimer",                             // name
+          "Test Data Calculation Time",                    // display name
+          "Processing time to find the test data",         // description
+          MetricType.TIMER,                                   // type
+          MetricUnits.NANOSECONDS);                           // units
+          testDataCalcTimer = registry.timer(testDataCalcTimerMetadata);
+  }
+    
+  /**
+    * getTestData - get the testing data
+    *
+    * @return JsonObject
+    */
+  public JsonObject getTestData() {
+      
+    // Start timing here
+    Timer.Context context = testDataCalcTimer.time();
+        
+    JsonObject value = Json.createObjectBuilder()
+    .add("firstName", "Jamie")
+    .add("lastName", "Coleman")
+    .add("age", 25)
+    .add("address", Json.createObjectBuilder()
+        .add("streetAddress", "1 Hurlsey Park")
+        .add("city", "Winchester")
+        .add("County", "Hampshire")
+        .add("postalCode", "SO17 1PR"))
+    .add("phoneNumber", Json.createArrayBuilder()
+        .add(Json.createObjectBuilder()
+            .add("type", "home")
+            .add("number", "212 555-1234"))
+        .add(Json.createObjectBuilder()
+            .add("type", "fax")
+            .add("number", "646 555-4567")))
+    .build();
+      
+      // Stop timing
+      context.close();
+      return value;
+    }
+}
+```
+
+4. Next we need to update the `LibertyRestEnpoint.java` which is located in the same folder as the `JaxrsManager.java` to look like the example below adding the @ApplicationScoped annotation which creates the object once for the duration of the application. The @Timed annotation denotes a timer, which tracks duration of the annotated object. We add these annotations so we can register the metrics for the endpoints in the file.

```

Comment:
This files does not exist! LibertyRestEnpoint.java

Reply:
This is the C:\work\tutorial-microprofile\microservice-one\src\main\java\application\rest\LibertyRestEndpoint.java not the one from the microservice-gateway.